app/code/Magento/StoreGraphQl/Model/Resolver/Store/StoreConfigDataProvider.php

```diff
@@ -55,23 +65,42 @@ public function __construct(
      */
     public function getStoreConfigData(StoreInterface $store): array
     {
-        return array_merge(
-            $this->getBaseConfigData($store),
-            $this->getExtendedConfigData((int)$store->getId())
-        );
+        $defaultStoreConfig = $this->storeConfigManager->getStoreConfigs([$store->getCode()]);
+        return $this->prepareStoreConfigData(current($defaultStoreConfig), $store->getName());
     }
 
     /**
-     * Get base config data
+     * Get available website stores
      *
-     * @param StoreInterface $store
+     * @param string $websiteId
      * @return array
      */
-    private function getBaseConfigData(StoreInterface $store) : array
+    public function getAvailableStoreConfig(string $websiteId): array
     {
-        $storeConfig = current($this->storeConfigManager->getStoreConfigs([$store->getCode()]));
+        $websiteStores = $this->storeWebsiteRelation->getWebsiteStores($websiteId, true);
```

Comment:
you need a (int) typecast, and plz do that int $websiteID in methods, it's the preferred way